lib/sms.php

```diff
@@ -112,34 +114,35 @@ function get_rest_url( $action ) {
  * @author Daniel Petrovic
  * @link http://www.php.net/manual/en/function.preg-split.php#108090
  */
-function split_words($longtext, $max = 1) {
+function split_words( $longtext, $max = 1 ) {
 	// spaces or commas are not considered to be words
 	// between '[' and ']' can be put all characters considered to be
 	// word separators
-	$words = preg_split('/[\s,]+/', $longtext, null, PREG_SPLIT_NO_EMPTY);
-	$add_line = false;
+	$words        = preg_split( '/[\s,]+/', $longtext, 0, PREG_SPLIT_NO_EMPTY );
```

Comment:
The documentation [says](https://www.php.net/preg_split) that `$limit` is `integer`, replacing `null` with `0`.